### PR TITLE
Update AVS intro page

### PR DESCRIPTION
**Pull request by dabit3**

### CODE REVIEW

**Comment by subtleGradient** [comment]
Please add a “What is AVS?” subheading that just literally explains what the acronym stands for and literally explains what this is. Is this a software product? Is this a standards proposal? Is this a conceptual paradigm?  Literally what is this thing and why should I read the rest of this page?

File: docs/eigenlayer/avs-guides/avs-developer-guide.md [hunk 1]
```diff
@@ -2,9 +2,28 @@
...
+
+The stake serves as a commitment to the users of the infrastructure when Restakers delegate their stake to Operators. The Operators earn incentives for proper verifiable operation of the AVS software and are subject to slashing in cases where they provably operate the AVS software maliciously.
+
+- **AVS Developers** are responsible for constructing the underlying infrastructure logic and software.
+
+- **Operators** are responsible for the correct operation of AVS software. They utilize the ETH delegated to them by restakers as a form of commitment to the users of the infrastructure. Operators earn incentives for proper verifiable operation of the AVS software and are subject to slashing in cases where they provably operate the AVS software maliciously.
+
+### What can be built?
+
+An AVS can be any proof of stake network including but not limited to sidechains, coprocessors, data availability layers, new virtual machines, keeper networks, oracle networks, bridges, threshold cryptography schemes, and trusted execution environments.
```
[the remark this answers is not in the document]
Comment (dabit3) [reply docs/eigenlayer/avs-guides/avs-developer-guide.md:21]: perfect, thank you!

File: docs/eigenlayer/avs-guides/avs-developer-guide.md [hunk 2]
```diff
@@ -2,9 +2,28 @@
...
+
```
[the remark this answers is not in the document]
Comment (dabit3) [reply docs/eigenlayer/avs-guides/avs-developer-guide.md:22]: yes 100%

**Review by wesfloyd — APPROVED** [review]
lgtm

**Pull request merged**